extensions/narayana-jta/deployment/src/main/java/io/quarkus/narayana/jta/deployment/NarayanaJtaProcessor.java

```diff
@@ -156,6 +170,11 @@ public void setProperties(NarayanaJtaRecorder recorder) {
             defaultProperties.remove(i);
         }
         recorder.setDefaultProperties(defaultProperties);
+        // This must be done before setNodeName as the code in setNodeName will create a TSM based on the value of this property
```

Comment:
Remark: this code hasn't actually been added to the PR, it was already present and it's a quirk of git that it's showing up as added code.